app/src/main/java/com/example/parkingsystem_kotlin/mvp/presenter/ParkingSizeDialogPresenter.kt

```diff
@@ -0,0 +1,24 @@
+package com.example.parkingsystem_kotlin.mvp.presenter
+
+import com.example.parkingsystem_kotlin.fragment.SpacesParkingDialogListener
+import com.example.parkingsystem_kotlin.mvp.contract.ParkingSizeDialogContract
+import com.example.parkingsystem_kotlin.util.Constants
+
+class ParkingSizeDialogPresenter(
+    private val view: ParkingSizeDialogContract.ParkingSizeView,
+) : ParkingSizeDialogContract.ParkingSizePresenter {
+
+    override fun setParkingSize(inputListener: SpacesParkingDialogListener) {
+        val parkingSize = view.getNewParkingSize()
+        if (parkingSize.isNotEmpty() && parkingSize.toLong() < Constants.PARKING_MAX_SIZE) {
```

Comment:
You could define this constant in the companion object

Reply:
Done !